### PR TITLE
Add the Symfony Flex base image

### PR DESCRIPTION
**Pull request by sroze**

### CODE REVIEW

**Comment by sroze** [comment]
@kierenevans @andytson-inviqa WDYT ? Should we actually merge with the Symfony image, not creating a flex one?

File: symfony-flex/usr/local/share/symfony/symfony_flex_functions.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    do_symfony_console doctrine:migrations:migrate
+  fi
+
+  HAS_DOCTRINE_ORM=$(has_package doctrine/orm)
+  if [ "$HAS_DOCTRINE_ORM" = "true" ]; then
+    do_symfony_console doctrine:schema:update --force
+  fi 
+}
+
+has_package() {
```
Comment (andytson-inviqa) [inline symfony-flex/usr/local/share/symfony/symfony_flex_functions.sh]: this could be put in the php images for general use

**Comment by continuouspipe-starter** [comment]
> as discussed, we think that there's no need for a separate flex image

@andytson-inviqa including the npx encore things? Should we put them in the Symfony base images?

**Comment by sroze** [comment]
> as discussed, we think that there's no need for a separate flex image

@andytson-inviqa including the npx encore things? Should we put them in the Symfony base images?

**Comment by sroze** [comment]
Oy oy oy! So.. what about this question? https://github.com/continuouspipe/dockerfiles/pull/329#issuecomment-330056736 :)

**Comment by sroze** [comment]
Closed in favour of #457

**Pull request closed without merging**